openbb_sdk/sdk/core/openbb_core/app/static/package/crypto.py

```diff
@@ -13,27 +12,34 @@
 
 
 class CLASS_crypto(Container):
+    """/crypto
+    load
+    """
+
+    def __repr__(self) -> str:
+        return self.__doc__ or ""
+
     @validate_arguments
     def load(
         self,
-        symbol: typing_extensions.Annotated[
+        symbol: Annotated[
```

Comment:
This breaks python 3.8

Reply:
ok, think that sdk always needs to be generated from python 3.8